components/youtube/new-videos.js

```diff
@@ -0,0 +1,65 @@
+const axios = require('axios')
+const youtube = {
```

Comment:
I'd recommend separating this into its own app file so that future sources can refer to that. Then you'll want to require that app file like we do in other sources.

Reply:
Created separate youtube.app.js file.